javascript/javascript-core/types/the-difference-between-null-undefined-and-nan.md

```diff
@@ -60,11 +60,7 @@ isNaN("Twenty Three") // true (it's not a number)
 What does the following code output?
 
 ```javascript
-function foo(num){
-  return num / 2;
-}
-
-console.log(foo("sum"));
+console.log("sum" / 2);
```

Comment:
@Stefan-Stojanovic what do you think about replacing "sum" with "1"?

```suggestion
console.log("1" / 2);
```

Reply:
It is better, I just tried to do a fast and simple change